src/librustc_llvm/Cargo.toml

```diff
@@ -12,8 +12,7 @@ crate-type = ["dylib"]
 [features]
 static-libstdcpp = []
 
-[dependencies]
-rustc_bitflags = { path = "../librustc_bitflags" }
+# FIXME this crate uses libc, which somehow works without a dependency here
```

Comment:
It's ok to remove these, crates in this crate graph currently don't explicitly depend on the std crate and below (so this is expected)